assets/css/contact.css

```diff
@@ -2,7 +2,9 @@
 /*....illustration and content....*/
 
 /** bymeacoffee**/
-
+body{
+    font-family: Montserat;
```

Comment:
add quotes on this then it will work

Reply:
It was working on my forked local version. I have changed and added quotes though as asked for.